Configuration/Routes.Service.yaml

```diff
@@ -79,8 +79,8 @@
   httpMethods: ['GET']
 -
   name: 'Get Policy Info'
-  uriPattern: 'get-policy-info'
+  uriPattern: 'get-lazy-node-data'
```

Comment:
I feel the naming is off here, we should probably try and find a better wording.

`get-extended-node-information` or `get-additional-node-metadata` or such?

Reply:
Same goes for all other isntances of `lazy` ofc.

Reply:
Yeah, I could come up with a good name quickly... `get-additional-node-metadata` sounds OK.